modules/saku-policy-store/src/saku/resolvers.clj

```diff
@@ -83,6 +89,29 @@
          (dal/upsert-identity-policies dal-obj)
          (mapv (partial util/transform-ab policy-map)))))
 
+(defn add-*-statements [dal-obj add-fn args]
+  (->> (:inputPolicy args)
+    (util/transform-ab policy-input-map)
+    (add-fn dal-obj)
+    (mapv (partial util/transform-ab policy-map))))
```

Comment:
Statement addition code is nearly identical, so we extract the shared impl here.